dascore/io/core.py

```
@@ -436,28 +472,40 @@ def _get_prioritized_list(self, input_type="file"):
         valid_fiberio_by_type = self._get_fiber_io_by_input_type(input_type)
         out = tuple(x for x in maybe_ios if x in valid_fiberio_by_type)
         # And return fiberIOs that much the input type.
+        self._lookup_cache[key] = out
         return out
 
-    @cached_method
     def load_plugins(self, format: str | None = None):
```

Ruff (0.15.21)

[error] Function argument `format` is shadowing a Python builtin

(A002)